### PR TITLE
Add a job to build CAPG node image periodically

### PR DESCRIPTION
**Pull request by dims**
Signed-off-by: Davanum Srinivas <davanum@gmail.com>

### CODE REVIEW

**Comment by cpanato** [comment]
I think this is good for now, I'm working in a job to build PRs and later the periodic for CAPG: https://github.com/kubernetes-sigs/image-builder/pull/445 that I need to debug and finish (planning to get back to that this weekend)

**Review by cpanato — APPROVED** [review]
/lgtm

for reference https://github.com/kubernetes-sigs/cluster-api-provider-gcp/pull/362

File: config/jobs/image-pushing/k8s-staging-cluster-api.yaml [hunk 1]
```diff
@@ -258,3 +258,53 @@ periodics:
...
+    serviceAccountName: gcb-builder
+    containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210426-51fd28e-master
+        env:
+        # TODO(dims): reuse this project for now to test this job.
+        # need to create a separate project.
+        - name: GCP_PROJECT
+          value: k8s-staging-cloud-provider-gcp
+        - name: SKIP_CLEANUP
+          value: true
```
[the remark this answers is not in the document]
Comment (dims) [reply config/jobs/image-pushing/k8s-staging-cluster-api.yaml]: gotcha!

File: config/jobs/image-pushing/k8s-staging-cluster-api.yaml [hunk 2]
```diff
@@ -258,3 +258,53 @@ periodics:
...
+      - image: gcr.io/k8s-testimages/image-builder:v20210302-aa40187
+        env:
+          # TODO(dims): reuse this project for now to test this job.
+          # need to create a separate project.
+          - name: "GCP_PROJECT"
+            value: "k8s-staging-cloud-provider-gcp"
+          - name: "SKIP_CLEANUP"
+            value: "true"
+        command:
+          - "runner.sh"
```
Comment (cpanato) [inline config/jobs/image-pushing/k8s-staging-cluster-api.yaml:292]: i think the runner.sh does not exist in the image-builder image

**Comment by cpanato** [comment]
Lets try!! We always learn something

/lgtm

**Pull request merged**